### ISSUE TITLE
5 Habits of Highly Unproductive People

### ISSUE THREAD

**Issue by jinglescode** [jinglescode/jinglescode.github.io#63]
# TL;DR
Getting more done in less time is an attainable goal if you don’t sabotage yourself

# Key Takeaways
- **Starting your day without a reliable routine**.A morning routine ensures you take action first thing in the morning. It’s essential to getting your mind into productive gear every day. Without some structure, your productivity can suffer. When you start your day without a routine, you can waste a lot of energy in the morning deciding what’s worth your time for the rest of the day.

- **Prioritising shallow work over deep work**. It is easy to get excited about low-value work — the shallow and easy tasks that keep us busy without really moving us forward: email, organising files, returning phone calls, meetings, paper work, etc. High-value tasks contribute to your long-term goals. Activities like executing the sales process, writing, designing, coding, and video or photo editing move the needle. They produce your most important outcomes.

- **Starting your day without a priority list**. To prioritise effectively you need to be able to recognise what is important and what is urgent. Getting real work done is about putting your time and energy where they are most needed when they are most needed.

- **Working without the right system**. All of these systems are designed to do one thing: get stuff done. The most popular productivity systems are: Getting Things Done (GTD), The SMART Method, Time-boxing, Biological Prime Time, Personal Kanban, The “To-Do” list, The “Must Do” list, Pomodoro Technique, Eisenhower Matrix, Don’t Break The Chain, and Eat That Frog.

- **Never truly disconnecting**. Quality rest from work doesn’t have to be huge to make a difference — just 5 or 15 minutes in between deep work can be enough. But the rest itself should be free from distractions or diversions — not filled with passive digital interaction.

# Link
https://medium.com/kaizen-habits/5-habits-of-highly-unproductive-people-5b8025757b88
